README.md

```diff
@@ -268,8 +268,8 @@ it('works with unit tests', () => {
 
 ### Redux Saga v0.15
```

Comment:
Maybe update this as well to something like "Redux Saga Support"

Reply:
Alright let me do it an PR